### ISSUE TITLE
A PyTorch reproduction of koncept512 code

### ISSUE THREAD

**Issue by ZhengyuZhao** [subpic/koniq#3]
Hi! Your work is interesting.

I just reproduce your Keras code for koncept512 model using PyTorch.
The new results (SRCC: 0.914 | PLCC: 0.929) are very close to what you have got (SRCC: 0.919 | PLCC: 0.932) in koncept512_train_test_py3.ipynb.

If you want, feel free to post the following link to my PyTorch reproduction into your official repository :-)
https://github.com/ZhengyuZhao/koniq-PyTorch

**Comment by subpic** [comment]
Thanks for sharing! :)

**Comment by zakajd** [comment]
@ZhengyuZhao  Thanks! You saved my time 😄

**Comment by ZhengyuZhao** [comment]
> @ZhengyuZhao Thanks! You saved my time 😄

Enjoy!

**Comment by Sissuire** [comment]
have you ever done the cross-dataset comparison (trained on KoN-10k, and tested on LIVE-Challenge) ?

**Comment by subpic** [comment]
Yes, please check the [https://arxiv.org/pdf/1910.06180.pdf](KonIQ-10k paper); Look for [LIVE-itw (in-the-wild)](http://live.ece.utexas.edu/research/ChallengeDB/), which is the LIVE-Challenge database.

**Comment by hulkds** [comment]
Hey the link for inceptionresnetv2 pretrained weights is die !

**Comment by ZhengyuZhao** [comment]
> Hey the link for inceptionresnetv2 pretrained weights is die !

Are you using this link: https://surfdrive.surf.nl/files/index.php/s/oeGv7wEyyMwwbIO?
It seems working for me...
